Fallout2/Fallout1in2/mods/fo1_german/text/german/dialog/OBJ_DUDE.MSG

```diff
@@ -12,63 +12,231 @@
+#
+#
 # HALLDED.MAP
 # The Necropolis Hall of the Dead Tell-Me-Abouts
+#
 {2800}{}{Meister}
-{2801}{}{Steuerchip}
-{2802}{}{Necropolis}
-{2803}{}{Set}
+{2801}{}{Steuerchip|Wasserchip|Chip}
+{2802}{}{Necropolis|Nekropolis|City of the Dead|City of Dead}
+{2803}{}{Set|Seth}
 {2804}{}{Wasserstelle}
-{2805}{}{Halle}
+{2805}{}{Halle|Halle der Toten|Totenhalle|Halle des Todes}
+{2806}{}{Hilfe|Hilf mir}
+{2807}{}{Doktor|Sanit�ter|Arzt}
+{2808}{}{Kaufen|Einkaufen|Verkaufen|Handel|Handeln|Gesch�ft|Laden}
+{2809}{}{Wasser}
+{2810}{}{Vault|Vault12|Vault 12|Vault-12|Bunker|Bunker12|Bunker 12|Bunker-12}
+{2811}{}{Wasteland|Wastes|Waste|Badland|Badlands|Waste Land}
+{2812}{}{Richtung|Richtungen|wo ist|wo hin|wohin}
+{2813}{}{Fallout|Fall out|nuklearer fallout}
+{2814}{}{Stehlen|Diebstahl|Dieb|Diebe|Verbrechen|Verbrecher|Kriminelle}
+{2815}{}{Hub}
+{2816}{}{Strahlung|Rads|Rad}
+{2817}{}{Hotel|Motel}
+{2818}{}{Supermutant|Supermutanten|Super Mutant|Super Mutanten|Mutant|Mutanten}
+{2819}{}{Brei|klump}
+{2820}{}{Brahmin|Brahmins|Brahma}
+{2821}{}{Job|Arbeit|Aufgabe|Geld|Geld verdienen}
+{2822}{}{Kanal|Kan�le|Abwasserkanal|Untergrund|Kanalisation}
+{2823}{}{Ghul|Ghuls}
+
 {2900}{}{Ist er nicht ein gro�er wichtiger Mann?}
 {2901}{}{Ich wei� nicht, aber die Wasserstelle hat vielleicht einen.}
 {2902}{}{Du bist mittendrin. Besser gesagt in dem, was davon �brig ist. Dieser Ort stirbt aus, langsam aber sicher.}
 {2903}{}{Oh, ich mag ihn, wirklich. Du findest ihn in der Halle.}
 {2904}{}{Es liegt im Norden. Pa�' auf die Wachen auf. Sie nehmen ihren Job wirklich ernst.}
 {2905}{}{Die Totenhalle. Wo Set arbeitet. Liegt mitten im Zentrum von Necropolis.}
+{2906}{}{Dort wirst du keine Hilfe finde.}
+{2907}{}{Du bist auf dich allein gestellt.}
+{2908}{}{Wir handeln nicht mit deiner Art.}
+{2909}{}{Von mir wirst du keines bekommen.}
+{2910}{}{Die meisten von uns stammen aus dieser Vault.}
+{2911}{}{Die Strahlung macht es gef�hrlich f�r Glatth�ute. Uns jedoch ist das egal.}
+{2912}{}{Wenn du durch die Stadt reisen willst, mu�t du durch die Abwasserkan�le gehen.}
+{2913}{}{Ist das wenn der Regen gl�ht?}
+{2914}{}{Wenn du erwischt wirst, schneiden sie dich auf.}
+{2915}{}{Wei� nicht, ist mir egal.}
+{2916}{}{Anders als Glatth�uten ist uns die Strahlung egal.}
+{2917}{}{�stlich von hier. Du kommst durch den Untergrund dahin.}
+{2918}{}{Die an der Wasserstelle? Die hau'n dich zu Brei.}
+{2919}{}{Ich habe gesehen, wie einmal einer der Mutanten eine Glatthaut zu klump geschlagen hat.}
+{2920}{}{Sicher, ich mag den Geschmack.}
+{2921}{}{Set hat vielleicht 'ne Aufgabe f�r dich.}
+{2922}{}{Die Kanalisation ist der einzige Weg an den meisten Tr�mmern der Stadt vorbei.}
+{2923}{}{Das sind wir.}
 
+#
 # HOTEL.MAP
 # The Necropolis Hotel Tell-Me-Abouts
+#
 {3400}{}{Meister}
-{3401}{}{Steuerchip}
-{3402}{}{Necropolis}
-{3403}{}{Set}
+{3401}{}{Steuerchip|Wasserchip|Chip}
+{3402}{}{Necropolis|Nekropolis|City of the Dead|City of Dead}
+{3403}{}{Set|Seth}
```

Comment:
`Set` and `Seth` are two different NPCs

Reply:
I know that, but people often missnaming them in my experience.